src/features/bivariate_color_manager/atoms/bivariateColorManagerResource.ts

```diff
@@ -241,15 +246,24 @@ export const bivariateColorManagerResourceAtom = createResourceAtom(
   true,
 );
 
-const fillLayersWithCorrelationLevel = (
+const fillLayersWithCorrelationLevelAndMaps = (
   bivariateColorManagerData: BivariateColorManagerData,
   numeratorCorellationMap: NumeratorCorellationMap,
+  axisNominatorInfo: AxisNominatorInfo,
 ): void => {
   Object.values(bivariateColorManagerData).forEach((row) => {
-    [...Object.values(row.horizontal), ...Object.values(row.vertical)].forEach(
-      (layer) => {
-        layer.correlationLevel = numeratorCorellationMap[layer.name] || 0;
-      },
-    );
+    let horizontalLayersSum = 0;
+    let verticalLayersSum = 0;
+
+    [...Object.values(row.horizontal)].forEach((layer) => {
```

Comment:
Why not just `Object.values(row.horizontal).forEach` ?

Reply:
Nice catch! It's just stayed after refactoring 🤦